frontend/src/components/5.Lobby/Lobby.js

```diff
@@ -0,0 +1,111 @@
+import {Link} from 'react-router-dom';
+import React, {useState} from 'react';
+import Help from '../Common/Help';
+import '../Common/Help.css';
+import '../5.Lobby/Lobby.css';
+
+const Lobby = () => {
+  const [ShareButtonPopup, setSharePopup] = useState(false);
+  const [helpButtonPopup, setHelpButtonPopup] = useState(false);
+
+  {/* link this to backend.
+  Backend just needs to push data into People and NumOfCusines array */}
+  let GroupCode = '';
+  GroupCode = 'HX8192';
```

Comment:
Is this so lint is happy since you are using GroupCode as a variable?

Reply:
yea since GroupCode is going to be changed to avoid making it const.

Reply:
oh I forgot to add in useEffect, I'll quickly add that in.